script/Common/00_00.txt

```diff
@@ -1202,28 +1208,29 @@ Sensei ran up to me and slapped me hard across the cheek.
 [sha:0b15ffceb43b82a99bdd36d31f6b3949f0f225f6]{
 -- Page 88, Offset 337.
 -- 先生は真剣な目で見つめてくる。
-Sensei looked at me with a stern gaze.
+Sensei looks at me with a stern gaze.
 }
 [sha:14d35a791f9c9a41a06d362ae33b221c99ca9fa1]{
 -- Page 88, Offset 338.
 -- 容赦のない目。わるい人間を罰する、揺るぎのない眼。
-Her eyes were unforgiving.
- A firm stare that condemned wrongdoers.
+Her eyes are unforgiving.
+ Her stare is firm, as to condemn any wrongdoers.
 }
 [sha:fc789f6380b2bb3b5318e2acc5d4453b297eb48d]{
 -- Page 89, Offset 339.
 -- ああ、なんてバカだったんだろう。
-Ah, what a fool I was.
+Ah, what a fool I am.
 }
 [sha:9a899b867a84f25e9483bd64370ebc92fce6f910]{
 -- Page 89, Offset 340.
 -- あんまりにもこの時間が楽しくて、
+// tense might be weiirdd
 I had been enjoying our time together so much―――
 }
 [sha:7cf878edab068a06963228d661ba9804b2dec1d8]{
 -- Page 89, Offset 341.
 -- 僕は自分のことを忘れていた。
-That I'd forgotten who I was.
+That I forgot who I am.
```

Comment:
revert this part to match with previous sentence. He had 'forgotten who he was' at the time but has remembered now.

> I had been enjoying our time together so much that I'd forgotten who I was.